iOS Client/PodcastsController/PodcastFilterViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  PodcastFilterViewController.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 4/9/18.
+//  Copyright © 2018 Mockingbird. All rights reserved.
+//
+
+import Foundation
+import ReSwift
+import UIKit
+
+class PodcastsFilterViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, StoreSubscriber {
+    
+    @IBOutlet weak var tableView: UITableView!
+    var visibleStreams: Set<PodcastStream> = Set<PodcastStream>()
+    var streams: [PodcastStream] = [.pz, .mockingCast, .mockingPulpit]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UINib(nibName: PodcastFilterTableViewCell.reuseIdentifier, bundle: nil), forCellReuseIdentifier: PodcastFilterTableViewCell.reuseIdentifier)
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.rowHeight = UITableViewAutomaticDimension
+        tableView.estimatedRowHeight = 110
+        tableView.tableFooterView = nil
```

Comment:
is this necessary?

Reply:
Yeah it prevents the table view from having trailing empty cells

Reply:
very strong, good to know